Patch to fix some trivial spelling errors

Index: cfengine2/src/client.c
===================================================================
--- cfengine2.orig/src/client.c	2012-03-20 22:36:20.000000000 +0000
+++ cfengine2/src/client.c	2012-03-20 22:46:35.000000000 +0000
@@ -52,7 +52,7 @@
  
 if (CONN->sd == CF_NOT_CONNECTED)
    {   
-   Debug("Opening server connnection to %s\n",server);
+   Debug("Opening server connection to %s\n",server);
 
    if (!RemoteConnect(server,ip->forceipv4,SHORT_CFENGINEPORT,STR_CFENGINEPORT))
       {
Index: cfengine2/doc/cfetoolcheck.8
===================================================================
--- cfengine2.orig/doc/cfetoolcheck.8	2012-03-20 22:45:20.000000000 +0000
+++ cfengine2/doc/cfetoolcheck.8	2012-03-20 22:45:23.000000000 +0000
@@ -148,7 +148,7 @@
 .IX Header "DESCRIPTION"
 Takes a new value and checks it against the averages currently in the database
 specified by \fIname\fR, located at the path specified by the \fB\-p\fR argument,
-or the current working directory if the \fB\-p\fR argument is ommitted. 
+or the current working directory if the \fB\-p\fR argument is omitted.
 The value will be associated with the current time, unless the \fB\-t\fR option is
 given. The output indicates how much higher or lower the new value is compared
 to the averages in the database, in terms of the number of standard deviations.
Index: cfengine2/doc/cfetoolcreate.8
===================================================================
--- cfengine2.orig/doc/cfetoolcreate.8	2012-03-20 22:45:20.000000000 +0000
+++ cfengine2/doc/cfetoolcreate.8	2012-03-20 22:45:23.000000000 +0000
@@ -153,7 +153,7 @@
 subdirectory of the current working directory.
 .PP
 The \fB\-d\fR, \fB\-w\fR, and \fB\-y\fR options specify the period over which the data will
-span, and may be combined to create multiple databases. If they are all ommitted,
+span, and may be combined to create multiple databases. If they are all omitted,
 the default is weekly.
 .SH "OPTIONS"
 .IX Header "OPTIONS"
Index: cfengine2/doc/cfetooldump.8
===================================================================
--- cfengine2.orig/doc/cfetooldump.8	2012-03-20 22:45:20.000000000 +0000
+++ cfengine2/doc/cfetooldump.8	2012-03-20 22:45:23.000000000 +0000
@@ -148,7 +148,7 @@
 for human readability. This format can be read by \fBcfetool import\fR.
 .PP
 The \fB\-d\fR, \fB\-w\fR, and \fB\-y\fR options specify which databases to include in the
-dump, and may be combined. If they are all ommitted, the default is to only
+dump, and may be combined. If they are all omitted, the default is to only
 dump the weekly database.
 .SH "OPTIONS"
 .IX Header "OPTIONS"
Index: cfengine2/doc/cfetoolimport.8
===================================================================
--- cfengine2.orig/doc/cfetoolimport.8	2012-03-20 22:45:20.000000000 +0000
+++ cfengine2/doc/cfetoolimport.8	2012-03-20 22:45:23.000000000 +0000
@@ -148,7 +148,7 @@
 \&\fBrrdtool dump\fR.
 .PP
 The \fB\-d\fR, \fB\-w\fR, and \fB\-y\fR options specify which databases to create and look
-for in the dump, and may be combined. If they are all ommitted, the default is
+for in the dump, and may be combined. If they are all omitted, the default is
 to only import the weekly database.
 .SH "OPTIONS"
 .IX Header "OPTIONS"
Index: cfengine2/doc/cfetoolupdate.8
===================================================================
--- cfengine2.orig/doc/cfetoolupdate.8	2012-03-20 22:45:20.000000000 +0000
+++ cfengine2/doc/cfetoolupdate.8	2012-03-20 22:45:23.000000000 +0000
@@ -149,7 +149,7 @@
 .IX Header "DESCRIPTION"
 Takes a new value and uses it to update the database specified by \fIname\fR,
 located at the path specified by the \fB\-p\fR argument, or the current working
-directory if the \fB\-p\fR argument is ommitted. The value will be associated with
+directory if the \fB\-p\fR argument is omitted. The value will be associated with
 the current time, unless the \fB\-t\fR option is given. In order for cfetool to
 function properly, this command should be issued at least once during each
 interval specified by the \fB\-\-step\fR option of \f(CW\*(C`cfetool create\*(C'\fR.
Index: cfengine2/src/cfagent.c
===================================================================
--- cfengine2.orig/src/cfagent.c	2012-03-20 22:45:45.000000000 +0000
+++ cfengine2/src/cfagent.c	2012-03-20 22:46:35.000000000 +0000
@@ -1846,7 +1846,7 @@
           break;
           
       case 'V': printf("GNU cfengine %s\n%s\n",VERSION,COPYRIGHT);
-          printf("This program is covered by the GNU Public License and may be\n");
+          printf("This program is covered by the GNU General Public License and may be\n");
           printf("copied free of charge.  No warranty is implied.\n\n");
           exit(0);
           
Index: cfengine2/src/cfenvd.c
===================================================================
--- cfengine2.orig/src/cfenvd.c	2012-03-20 22:45:50.000000000 +0000
+++ cfengine2/src/cfenvd.c	2012-03-20 22:46:35.000000000 +0000
@@ -120,7 +120,7 @@
          break;
 
       case 'V': printf("GNU %s-%s daemon\n%s\n",PACKAGE,VERSION,COPYRIGHT);
-         printf("This program is covered by the GNU Public License and may be\n");
+         printf("This program is covered by the GNU General Public License and may be\n");
          printf("copied free of charge. No warrenty is implied.\n\n");
          exit(0);
          break;
Index: cfengine2/src/cfexecd.c
===================================================================
--- cfengine2.orig/src/cfexecd.c	2012-03-20 22:45:53.000000000 +0000
+++ cfengine2/src/cfexecd.c	2012-03-20 22:46:35.000000000 +0000
@@ -148,7 +148,7 @@
          break;
 
       case 'V': printf("GNU %s-%s daemon\n%s\n",PACKAGE,VERSION,COPYRIGHT);
-          printf("This program is covered by the GNU Public License and may be\n");
+          printf("This program is covered by the GNU General Public License and may be\n");
           printf("copied free of charge. No warrenty is implied.\n\n");
           exit(0);
           break;
Index: cfengine2/src/cfkey.c
===================================================================
--- cfengine2.orig/src/cfkey.c	2012-03-20 22:45:56.000000000 +0000
+++ cfengine2/src/cfkey.c	2012-03-20 22:46:35.000000000 +0000
@@ -252,7 +252,7 @@
           break;
           
       case 'V': printf("GNU cfengine %s\n%s\n",VERSION,COPYRIGHT);
-          printf("This program is covered by the GNU Public License and may be\n");
+          printf("This program is covered by the GNU General Public License and may be\n");
           printf("copied free of charge.  No warranty is implied.\n\n");
           exit(0);
           
Index: cfengine2/src/cfservd.c
===================================================================
--- cfengine2.orig/src/cfservd.c	2012-03-20 22:46:01.000000000 +0000
+++ cfengine2/src/cfservd.c	2012-03-20 22:46:35.000000000 +0000
@@ -210,7 +210,7 @@
          break;
 
       case 'V': printf("GNU %s-%s daemon\n%s\n",PACKAGE,VERSION,COPYRIGHT);
-          printf("This program is covered by the GNU Public License and may be\n");
+          printf("This program is covered by the GNU General Public License and may be\n");
           printf("copied free of charge. No warrenty is implied.\n\n");
           exit(0);
           break;
Index: cfengine2/src/cfshow.c
===================================================================
--- cfengine2.orig/src/cfshow.c	2012-03-20 22:46:09.000000000 +0000
+++ cfengine2/src/cfshow.c	2012-03-20 22:46:35.000000000 +0000
@@ -199,7 +199,7 @@
          break;
 
       case 'V': printf("GNU %s-%s db tool\n%s\n",PACKAGE,VERSION,COPYRIGHT);
-          printf("This program is covered by the GNU Public License and may be\n");
+          printf("This program is covered by the GNU General Public License and may be\n");
           printf("copied free of charge. No warrenty is implied.\n\n");
           exit(0);
           break;
